drivers/serial/Kconfig.nrfx

```diff
@@ -357,23 +80,4 @@ config NRFX_TIMER4
 		|| UART_2_NRF_HW_ASYNC_TIMER = 4 \
 		|| UART_3_NRF_HW_ASYNC_TIMER = 4
 
-
-config UARTE_NRF_HW_ASYNC
-	def_bool y
-	depends on UART_0_NRF_HW_ASYNC \
-		|| UART_1_NRF_HW_ASYNC \
-		|| UART_2_NRF_HW_ASYNC \
-		|| UART_3_NRF_HW_ASYNC
-	select NRFX_PPI if HAS_HW_NRF_PPI
-	select NRFX_DPPI if HAS_HW_NRF_DPPIC
```

Comment:
How are these options supposed to be enabled now?

Reply:
it's moved to the template.

Reply:
ah, right, I somehow missed that